doc/user-guide/04-scenarios.adoc

```diff
@@ -67,6 +67,103 @@ a single confidential command to let the redirection also apply for 'set -x'.
 See the description in /usr/share/rear/conf/default.conf
 how to set variables for secret values in a confidential way.
 
+== Booting ReaR rescue initramfs on IBM Z (s390x) under z/VM
+
+There are two options how to boot the rescue environment on IBM Z (s390x) under
+z/VM created using the `OUTPUT=RAMDISK` setting.
+
+1. Using the `kexec` tool as described in link:../../usr/share/rear/conf/templates/RESULT_usage_RAMDISK.txt[RESULT_usage_RAMDISK.txt],
+   if you have access to a running Linux instance.
+
+2. Using the `zipl` tool for management of boot devices on s390/s390x.
+
+=== Booting ReaR rescue initramfs on IBM Z (s390x) under z/VM with `zipl`
+
+After creating the rescue image with `rear mkrescue`, move the kernel and
+initramfs to the target directory (denoted by `REAR_DIRECTORY` in this example)
+on the target device to boot from.
+
+NOTE: See `zipl(8)` for all device types supported by the `--target` option.
+      If you want to use a tape device, see `zipl(8)` to use the `--tape`
+      option instead. Note that the `--tape` option with ReaR is untested so
+      contributions to this guide are welcome! Also, it is not possible to use
+      `zipl` with the z/VM reader device.
+
+IMPORTANT: The following method will overwrite the boot configuration on a device
+           which contains the given `--target` directory!
+
+Use the following command to make the device with `REAR_DIRECTORY` bootable
+with `zipl`:
+```
+# zipl --target=BOOTMAP_INSTALL_DIRECTORY \
+       --parameters='root=/dev/ram0 ro' \
+       --image=REAR_DIRECTORY/kernel-HOSTNAME \
+       --ramdisk=REAR_DIRECTORY/initramfs-HOSTNAME.img
+Run /lib/s390-tools//zipl_helper.device-mapper BOOTMAP_INSTALL_DIRECTORY
+Building bootmap in 'BOOTMAP_INSTALL_DIRECTORY'
+Adding IPL section
+  initial ramdisk...: REAR_DIRECTORY/initramfs-HOSTNAME.img
+  kernel image......: REAR_DIRECTORY/kernel-HOSTNAME
+  kernel parmline...: 'root=/dev/ram0'
+  component address:
+    internal loader.: 0x0000a000-0x0000dfff
+    parameters......: 0x00009000-0x00009fff
+    kernel image....: 0x00010000-0x00cdbfff
+    parmline........: 0x00cdc000-0x00cdcfff
+    initial ramdisk.: 0x00cf0000-0x10980fff
+    environment blk.: 0x00cec000-0x00cecfff
+Preparing boot device for CCW- and LD-IPL: dasda.
+Done.
+```
+You can set `BOOTMAP_INSTALL_DIRECTORY` to any directory on the same device
+with `REAR_DIRECTORY` to store the boot data. The bootloader in the example was
```

Comment:
Sorry if I am rather nitpicking here
but I still do not fully understand things.

Here it looks as if BOOTMAP_INSTALL_DIRECTORY
is restricted by REAR_DIRECTORY so if for example
REAR_DIRECTORY is on a device that is not meant
to be booted from then the user may falsely set
BOOTMAP_INSTALL_DIRECTORY to a directory on that device.

I think the restiriction (if it exisist at all, see below)
is the other way round:
If /dev/dasda should be used to boot from
then BOOTMAP_INSTALL_DIRECTORY must be on /dev/dasda
and also REAR_DIRECTORY must be on /dev/dasda.

But perhaps only BOOTMAP_INSTALL_DIRECTORY
must be on /dev/dasda while REAR_DIRECTORY could
be on any device?

I fail to see in
https://manpages.debian.org/jessie/s390-tools/zipl.8
if REAR_DIRECTORY must be on the same disk device
where BOOTMAP_INSTALL_DIRECTORY is.

Simply put:
I do not understand if REAR_DIRECTORY is only a
"source" directory where zipl takes kernel and initrd from
and copies them (always of perhaps only if needed)
to the target destination on the boot disk that is
implicitly specified by BOOTMAP_INSTALL_DIRECTORY
OR
if kernel and initrd stay in REAR_DIRECTORY so that
REAR_DIRECTORY must be also on the boot disk that is
implicitly specified by BOOTMAP_INSTALL_DIRECTORY ?

Reply:
> kernel and initrd stay in REAR_DIRECTORY so that
REAR_DIRECTORY must be also on the boot disk that is
implicitly specified by BOOTMAP_INSTALL_DIRECTORY

This is what happens.  (The manual page does not say that but as noted in (2) https://github.com/rear/rear/pull/3158#issuecomment-1964207577, `zipl(8)` won't necessarily stop you to use different devices, but the machine will not boot!)

I've reworded the section to make it more explicit.